new/LIT60499HymnMarySalotki.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT60499HymnMarySalotki" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Hymn to Virgin Mary</title>
+                <title xml:lang="gez" xml:id="t2">ኦእግዝእትየ፡ ቅድስት፡ ድንግል፡ በክልኤ፡ ማርያም፡ ጸሎትኪ፡ ወባራካትኪ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">ʾO-ʾǝgzǝʾtǝya qǝddǝst dǝngǝl ba-kǝlʾe Māryām ṣalotǝki wa-bārākātǝki ...</title>
```

Comment:
I am not completely sure whether this should be interpreted as an individual hymn or an elaborate supplication to the introductive exhortation. What do you think, @DenisNosnitsin1970 ? in ESum014
In any case: "...to the Virgin"... and  barakatǝki (I checked the MS)

Reply:
I do not know. It might be as you say, but it might be well different, but then another witness with this piece would be necessary, and a look in the editions. At these point I must say that all these records are so inexplicit, with hidden folio numbers and other information - finding of the text mentioned took me as least 15 mins, I am sorry. In the currect version of the DB, the description is also incredibly  fragmented and untransparent. There is no way to come to the needed item except then throug xml.

Reply:
Well these are records which I created for the latest batch you just approved ;) it is not difficult to find this one in UM-014

Reply:
In the editions that I checked this part is not present. 
The current presentation is due to the complicated history of these records. At least Zhenia is making them to some point retrievable, though of course it is still far from perfect. As we all know, this would necessitate a very considerable time investment, which we probably don't have...
Though I must say that to me the pull requests with many manuscripts are a bit unhandy. I would prefer one for one or two manuscripts. But this is a topic on which any three people will have nine opinions :)